### PR TITLE
support sending /api/v1/series queries

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: Ben Ye <yb532204897@gmail.com>

Example:

``` yaml
queries:
- name: series
  matchers:
  - '{instance!=""}'  
  duration: 1w
  cache: false
```

### CODE REVIEW

File: cmd/up/main.go [hunk 1]
```diff
@@ -529,9 +534,17 @@ func parseQueriesFileName(opts *options.Options, l log.Logger, queriesFileName s
...
-			_, err = parser.ParseExpr(q.Query)
-			if err != nil {
-				return fmt.Errorf("query %q in --queries-file content is invalid: %w", q.Name, err)
+			if len(q.Matchers) > 0 {
+				for _, s := range q.Matchers {
+					if _, err := parser.ParseMetricSelector(s); err != nil {
+						return fmt.Errorf("query %q in --queries-file matchers are invalid: %w", q.Name, err)
+					}
+				}
+			} else if q.Query != "" {
```
Comment (bwplotka) [inline cmd/up/main.go:543]: No need for else we can use `continue`

**Review by bwplotka — APPROVED** [review]
LGTM, just one point (:

File: cmd/up/main.go (hunk 1 above)
Comment (yeya24) [reply cmd/up/main.go]: Good catch! Updated

**Review by kakkoyun — APPROVED** [review]
🥇 LGTM

**Pull request merged**